packages/jsii-pacmak/lib/packaging.ts

```diff
@@ -52,15 +55,20 @@ export class JsiiModule {
   /**
    * Prepare an NPM package from this source module
    */
-  public async npmPack() {
+  public async npmPack(packCommand = DEFAULT_PACK_COMMAND) {
     this._tarball = await Scratch.make(async (tmpdir) => {
-      // Quoting (JSON-stringifying) the module directory in order to avoid
-      // problems if there are spaces or other special characters in the path.
-      const args = ['pack', JSON.stringify(this.moduleDirectory)];
+      const args = packCommand.split(' ');
+
+      // If using the default command, optimize by packing directly into the tmp dir to allow unit tests to run in parallel
+      if (packCommand === DEFAULT_PACK_COMMAND) {
+        args.push('--pack-destination', tmpdir);
+      }
       if (logging.level >= logging.LEVEL_VERBOSE) {
         args.push('--loglevel=verbose');
       }
```

Comment:
I think this should not happen unless using the default command (the user has control, so they can decide on log level there directly).